sycl/test/basic_tests/stream/no_intrinsic.cpp

```diff
@@ -0,0 +1,32 @@
+// RUN: %clangxx -S -emit-llvm -fsycl-device-only %s -o - -Xclang -disable-llvm-passes | FileCheck %s
+
+//==------------------ no_intrinsics.cpp - SYCL stream test ----------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// Test to check that intrinsics to get a global id are not generated for the
+// stream.
+
+// CHECK-NOT: call spir_func void @{{.*}}__spirvL22initGlobalInvocationId{{.*}}
```

Comment:
Nit. I think you should check that there's no call to spirv initGlobalInvocationId *inside the kernel*. That means, that the kernel should still be there.

Reply:
I didn't get this comment. This check covers all call instructions i.e. inside the kernel as well.

Reply:
What I mean is that if the frontend doesn't generate this instruction for some reason we have to be sure that the kernel is still generated. I believe that there are proper checks for that in frontend part of tests.